packages/@n8n/benchmark/Dockerfile

```diff
@@ -33,7 +33,7 @@ COPY --chown=node:node ./packages/@n8n/benchmark/package.json /app/packages/@n8n
 COPY --chown=node:node ./patches /app/patches
 COPY --chown=node:node ./scripts /app/scripts
 
-RUN pnpm install --frozen-lockfile
+RUN pnpm install --frozen-lockfile --ignore-scripts=true
```

Comment:
won't this also skip downloading the sqlite binary?

Reply:
Yeah probably. Will take another approach